pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -776,7 +823,20 @@ class CollectionVersionViewSet(
 
     lookup_field = "version"
 
-    DEFAULT_ACCESS_POLICY = _PERMISSIVE_ACCESS_POLICY
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            _CAN_VIEW_REPO_CONTENT,
+            {
+                "action": ["destroy"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_model_or_obj_perms:ansible.delete_collection",
+                    "has_model_or_obj_perms:ansible.view_collection",
```

Comment:
This viewset is for CollectionVersions which is a content type that we don't assign permissions to. So this probably needs a special condition method to find the Collection object that the CollectionVersion is a part of and then check the user's perms on that Collection.